airbyte-integrations/connectors/destination-s3/Dockerfile

```diff
@@ -15,6 +15,32 @@ WORKDIR /airbyte
 ENV APPLICATION destination-s3
 
 COPY --from=build /airbyte /airbyte
+RUN /bin/bash -c 'set -ex && \
```

Comment:
do we need the `-x` set here? I know in the past it led to a security incident because the debug logs were logging docker tokens

Reply:
> do we need the `-x` set here? I know in the past it led to a security incident because the debug logs were logging docker tokens

@Phlair removed  `-x` flag